src/common/utils.h

```diff
@@ -0,0 +1,20 @@
+#ifndef GFS_COMMON_UTILS_H_
+#define GFS_COMMON_UTILS_H_
+
+#include "google/protobuf/stubs/status.h"
+#include "grpcpp/grpcpp.h"
+
+namespace gfs {
+namespace common {
+namespace utils {
```

Comment:
Acknowledged.
My experience is it's easier to have it as functions, rather than classes, when you don't have many utility functions. Once we have more utility functions, then it makes sense to group them into utility classes, which can be done later. It's an overkill for now.